types/node-cron/index.d.ts

```diff
@@ -15,7 +16,6 @@ export function validate(cronExpression: string): boolean;
 export interface ScheduledTask {
     start: () => this;
     stop: () => this;
-    destroy: () => void;
     getStatus: () => string;
```

Comment:
getStatus missing too https://github.com/node-cron/node-cron/blob/fbc403930ab3165ffef7d53387a29af92670dfea/src/scheduled-task.js#L7

Reply:
@aseidma could you please revise?

Reply:
Will revise later today!